File: ejercicio7/rectangulo.php
<?php
class Rectangulo{

    public $base;
    public $altura;

    public function calcularArea(){

        return $this->base * $this->altura;
    }
}

$rectangulo = new Rectangulo();
$rectangulo->base = 10;
$rectangulo->altura = 5;

echo $rectangulo->calcularArea();

?>

File: Ejercicio5/EJ 53/src/Rectangulo.java

public class Rectangulo extends Figura {

    double base;
    double altura;
    
    public Rectangulo(double base, double altura) {
        this.base = base;
        this.altura = altura;
    }
	
	@Override
    double calcularArea() { //Calcula el área del rectángulo
        return base * altura;
    }
}
